db/migrate/20170222214902_add_initiator_to_service.rb

```diff
@@ -0,0 +1,13 @@
+class AddInitiatorToService < ActiveRecord::Migration[5.0]
+  class Service < ActiveRecord::Base; end
+  def up
+    add_column :services, :initiator, :string
```

Comment:
It can't hurt.

@mkanoor Note that you can put the comment on the add_column call.